pylxd/models/network.py

```diff
@@ -17,18 +17,27 @@
 class Network(model.Model):
     """A LXD network."""
     name = model.Attribute()
+    description = model.Attribute()
     type = model.Attribute()
-    used_by = model.Attribute()
     config = model.Attribute()
-    managed = model.Attribute()
+    managed = model.Attribute(readonly=True)
+    used_by = model.Attribute(readonly=True)
+
+    @classmethod
+    def exists(cls, client, name):
+        """Determine whether a network exists."""
```

Comment:
Please can we start documenting the types in the docstrings.  For this one it would be:

```
    """Ask LXD if the network called <name> exists.

    :param client: pylxd client instance
    :type client: pylxd.Client()
    :param name: the name of the network
    :type name: str
    :returns: True if the network exists
    :rtype: Boolean
    """
```

This needs to be done for each of the public facing API calls.  The reason for this is that we will be eventually transition the documentation over to  the API docstrings, and remove a lot of the 'hand written' documentation.

Reply:
I really like this idea - my IDE will be very happy about this. I am using [PEP 526](https://www.python.org/dev/peps/pep-0526/) but it's fairly new improvement and it's not compatible with older Python versions

Reply:
Yes, unfortunately, PEP526 is a Python 3.6-ism, and this library needs to maintain compatibility with Py27 for the moment.